app/api/api_v1/routers/ingest.py

```diff
@@ -31,24 +31,26 @@ def get_event_template():
     event_schema = IngestEventDTO.model_json_schema(mode="serialization")
     event_template = event_schema["properties"]
 
-    del event_template["family_document_import_id"]
-    del event_template["family_import_id"]
-
     return event_template
 
 
-def get_document_template():
+def get_document_template(corpus_type: str):
     document_schema = IngestDocumentDTO.model_json_schema(mode="serialization")
     document_template = document_schema["properties"]
-
-    del document_template["family_import_id"]
-    document_template["events"] = [get_event_template()]
+    document_template["metadata"] = get_metadata_template(corpus_type, "document")
 
     return document_template
 
 
-def get_metadata_template(corpus_type: str):
-    return taxonomy.get(corpus_type)
+def get_metadata_template(corpus_type: str, metadata_type: str):
+    metadata = taxonomy.get(corpus_type)
+    if not metadata:
+        return {}
+    if metadata_type == "document":
+        return metadata.pop("_document")
```

Comment:
Could we use an Enum here? (see db-client [EntitySpecificTaxonomyKeys](https://github.com/climatepolicyradar/navigator-db-client/blob/c1d6e7f8fead91df449648307550b9a4ad63a45b/db_client/models/dfce/taxonomy_entry.py#L29))